pyproject.toml

```diff
@@ -102,6 +102,8 @@ testpaths = [
 filterwarnings = [
     "error",
     "ignore::ResourceWarning",
+    # chromadb
+    "ignore:Python 3.14 will:DeprecationWarning",
```

Comment:
I guess we never even hit this issue in #563 as it was shadowed by other things. We need to re-add it.